kitsune/community/static/less/community-new.less

```diff
@@ -6,37 +6,73 @@
     }
 }
 
-table.top-contributors {
-    width: 100%;
+.SelectTable {
+    table.top-contributors {
+        width: 100%;
+
+        thead {
+            tr {
+                th {
+                    padding-bottom: 5px;
+                }
+            }
+        }
 
-    thead {
-        tr {
-            th {
-                padding-bottom: 5px;
+        tbody {
+            tr {
+                height: 40px;
+
+                td {
+                    background: #fff;
+                    border-bottom: 2px solid #EAEFF2;
+                    min-width: 25px;
+                }
             }
         }
-    }
 
-    tbody {
-        tr {
-            height: 40px;
+        [data-column="select"] {
+            text-align: center;
+        }
+
+        [data-column="actions"] {
+            padding-right: 10px;
+            text-align: center;
+            width: 0; // As small as possible.
```

Comment:
Yeah. This is actually an old line, I just re-indented a lot of stuff here. Sorry about that.